cpp/ql/lib/semmle/code/cpp/models/interfaces/Throwing.qll

```diff
@@ -11,12 +11,21 @@ import semmle.code.cpp.models.Models
 import semmle.code.cpp.models.interfaces.FunctionInputsAndOutputs
 
 /**
- * A class that models the exceptional behavior of a function.
+ * A function that is known to raise an exception.
```

Comment:
I would leave this as it was, as we're going to deprecate this anyway.

Reply:
Fixed